pkg/build/tar.go

```diff
@@ -39,8 +40,13 @@ func addFileToTarWriter(filePath string, tarWriter *tar.Writer) error {
 
 	header := &tar.Header{
 		Typeflag: tar.TypeDir,
-		Name:     "disk",
+		Name:     "disk/",
 		Mode:     0555,
+		Uid:      107,
+		Gid:      107,
+		Uname:    "qemu",
+		Gname:    "qemu",
+		ModTime:  time.Now(),
 	}
```

Comment:
Yes, otherwise it is in principle up to the untar procedure to determine ownership on the directory and permissions and this may or may not work for us then.